components/rating/rating.css

```diff
@@ -0,0 +1,9 @@
+.rating-container{
+background-color:white;
+}
+.rating-star-active{
+color: var(--secondary)
+}
+.fa-star{
+    cursor:pointer
```

Comment:
Suggestion : try to have ```;``` everytime a style is given and it ends

Reply:
yeah it is good practice and will also help if you plan to transpile your code at some later point of time.